# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: mozglue/tests/gtest/moz.build
# Commit: 670256c85479
# Full Hash: 670256c85479f9ae966ff6ab661106606150f2cb
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-10-18 21:14:52
# Regressor Bug: 1717448
# File Overlap Count: 1
# Description:
#   Bug 1717448 - Add some tests for Gecko integration. r=nika,glandium
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217299
# ==============================================================================

diff -r 37ab78b00a4f -r 670256c85479 mozglue/tests/gtest/moz.build
--- a/mozglue/tests/gtest/moz.build	Fri Oct 18 11:35:46 2024 +0000
+++ b/mozglue/tests/gtest/moz.build	Fri Oct 18 11:35:51 2024 +0000
@@ -2,12 +2,13 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+SOURCES += ["TestFmt.cpp"]
+
 if CONFIG["OS_ARCH"] == "WINNT":
     SOURCES += [
         "TestNativeNtGTest.cpp",
     ]
 
-
 if not CONFIG["MOZ_TSAN"]:
     SOURCES += [
         "TestStackWalk.cpp",
